app/bundles/CampaignBundle/Command/ExportCampaignCommand.php

```diff
@@ -0,0 +1,287 @@
+<?php
+
+namespace Mautic\CampaignBundle\Command;
+
+use Mautic\CoreBundle\Command\ModeratedCommand;
+use Mautic\CoreBundle\Helper\CoreParametersHelper;
+use Mautic\CoreBundle\Helper\PathsHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ExportCampaignCommand extends ModeratedCommand
```

Comment:
Please make new classes final by default. It shouldn't be final only for a good reason.
```suggestion
final class ExportCampaignCommand extends ModeratedCommand
```

Reply:
fixed